providers/openshift_create_pv.rb

```diff
@@ -24,20 +24,20 @@ def whyrun_supported?
         'server_pv' => pv['server'],
         'volume_policy' => pv.key?('policy') ? pv['policy'] : 'Retain'
       )
-      not_if '[[ `oc get pv/${name_pv} --no-headers --config=admin.kubeconfig | wc -l` -eq 1 ]]'
+      not_if "[[ `oc get pv/${name_pv} --no-headers --config=#{node['cookbook-openshift3']['openshift_master_config_dir']}/admin.kubeconfig | wc -l` -eq 1 ]]"
     end
 
     next unless pv.key?('claim')
     execute "Create Persistent Claim: #{pv['name']}" do
       cwd node['cookbook-openshift3']['openshift_master_config_dir']
```

Comment:
Hi @ianmiell , do you know if the `cwd` attribute of execute resource applies to the not_if/only_if guard clauses? If not, then yes we have to explicit the admin.kubeconfig path. I'll check it out.